check_vmware_api.pl

```diff
@@ -377,6 +377,14 @@ sub main {
 		. '   Virtual machine name.',
 		required => 0,
 	);
+	
+	$np->add_arg(
+		spec => 'port|P=i',
+		help => "-P, --port=<port>\n"
+		. "   Port to connect to.\n"
+		. '   By default, port 443 will be used.',
+		required => 0,
```

Comment:
Here you can add the default value with "default => 443," then you do not need to test for it on line 551.